### DISCUSSION TITLE
映射怎样没有自动拉取远程数据

### DISCUSSION THREAD

**Discussion by VencentXI** [baidu/amis#6710] [General]
映射怎样没有自动拉取远程数据，请教一下问什么

**Comment by KillenMr** [comment]
映射具体指的什么

**Comment by VencentXI** [comment]
这是来自QQ邮箱的自动回复邮件。 你好！你的邮件我已经收到，我会尽快查看。

**Comment by wencan** [comment]
意思是：初始化时，不自动拉取数据？
api 的sendOn可以实现： https://baidu.github.io/amis/zh-CN/docs/types/api?page=1#%E9%85%8D%E7%BD%AE%E8%AF%B7%E6%B1%82%E6%9D%A1%E4%BB%B6

**Comment by VencentXI** [comment]
这是来自QQ邮箱的自动回复邮件。 你好！你的邮件我已经收到，我会尽快查看。
